### DISCUSSION TITLE
posts/tutorial1/

### DISCUSSION THREAD

**Discussion by giscus** [7w1/7w1.github.io#6] [Announcements]
# posts/tutorial1/

Creating a new game

https://7w1.github.io/posts/tutorial1/

**Comment by titusgroen** [comment]
Any chance of a tutorial on the Windows Theme as well?

**Reply by 7w1** [reply]
The windows theme I’m using is catppuccin. I have since switched my windows setup so probably no tutorial, but I linked the theme on the Aurora 4x discord server a while ago (I think) and there’s plenty of YouTube videos for installing it, you can google around.
